ApiBundle/Controller/ContentTypeController.php

```diff
@@ -138,6 +138,9 @@ public function deleteAction($contentTypeId)
     {
         if (0 == $this->get('open_orchestra_model.repository.content')->countByContentType($contentTypeId)) {
             $contentTypes = $this->get('open_orchestra_model.repository.content_type')->findBy(array('contentTypeId' => $contentTypeId));
+            foreach ($contentTypes as $contentType) {
```

Comment:
That's what is done. The repository send all version of contentType as the selection is done on contentTypeId and not id.